modules/ztp-validating-the-generation-of-installation-crs.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/ztp-zero-touch-provisioning.adoc
+
+[id="ztp-validating-the-generation-of-installation-crs_{context}"]
+= Validating the generation of installation CRs
+
+SiteConfig applies Installation custom resources (CRs) to the hub cluster in a namespace with the name matching the site name. To check the status, use the following command:
```

Comment:
s/SiteConfig/`SiteConfig`

The check status step should be moved under procedure. 

"To check the status, enter the following command:"

Reply:
changed  qand 2. Did not move the check status step.